src/parser.mly

```diff
@@ -0,0 +1,142 @@
+/* Ocamlyacc parser for SIFT */
+%{ 
+  open Ast 
+%}
+
+/* Arithmetic Operators */
+%token ADD SUBTRACT MULTIPLY DIVIDE MOD
+/* Relational Operators */
+%token EQ LEQ GT GEQ NEQ LT
+/* Logical Operators */
+%token AND OR NOT XOR
+/* Paranthesis and Brackets */
+%token LPAREN RPAREN LBRACE RBRACE LARROW RARROW LSQBRACE RSQBRACE
+/* Delimiters */
+%token COMMA SEMICOLON COLON DOT
+/* Assignment */
+%token ASSIGN 
+/* Features */
+%token PIPE
+/* Keywords */
+%token IF ELIF ELSE SWITCH CASE FOR WHILE CONTINUE BREAK DEFAULT LAMBDA DEFINE EXIT RETURN NULL TRUE FALSE
+%token IMPORT LIST FROM PURE 
+/* Primitive Data Types */
+%token INT FLOAT CHAR SYMBOL STRING BOOL
```

Comment:
I recommend that `SYMBOL` be replaced with `SYM` and `STRING` with `STR` to match what we use in the code

Reply:
Okay. Will do that